test/test_hawkmoth.py

```diff
@@ -9,7 +9,9 @@
 from hawkmoth.parser import parse_to_string
 
 def _get_output(input_filename, **options):
-    return parse_to_string(input_filename, False, **options)
+    output, errors = parse_to_string(input_filename, False, **options)
+    # FIXME: should handle errors
+    return output
```

Comment:
Yup, that's ok for now if @jnikula agrees to postpone the testing side of things.